tests/integration/test_vulnerability_detector/test_providers/test_providers_no_os.py

```diff
@@ -57,115 +57,153 @@
     - providers
 '''
 import os
-
 import pytest
+
+from wazuh_testing.modules.vulnerability_detector import event_monitor as evm
+from wazuh_testing.db_interface import cve_db
 from wazuh_testing.modules import vulnerability_detector as vd
-from wazuh_testing.tools import LOG_FILE_PATH
-from wazuh_testing.tools.configuration import load_wazuh_configurations, check_apply_test
-from wazuh_testing.tools.monitoring import FileMonitor
 from wazuh_testing.tools.services import control_service
-# from wazuh_testing.vulnerability_detector import make_vuln_callback, VULN_DETECTOR_EXTENDED_GLOBAL_TIMEOUT
-# Disabled until refactor
-
+from wazuh_testing.tools import configuration as config
```

Comment:
Why you have used this alias?

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/2547/commits/a6bca2e3641a2ab812ce3de8bfa09045b5fcd4e9